### ISSUE TITLE
Departments field only show the ID, not their name on publication details page

### ISSUE THREAD

**Issue by netsensei** [ugent-library/biblio-backend#38]
**Problem description**

On the publication detail page, the "departments" fieldset shows a list of departments associated with the publication. Currently, only the ID is shown to the end user. This doesn't convey a whole lot of information as to what department is associated with the publication.

**Cause**

This is what we get from the REST API:

```
      "department": [
         {
            "tree": [
               {
                  "_id": "GUK"
               }
            ],
            "_id": "KR71"
         },
         {
            "tree": [
               {
                  "_id": "LA"
               },
               {
                  "_id": "UGent"
               }
            ],
            "_id": "LA61"
         }
      ],
```

**Possible solutions**

* Consider expanding the object saved in Librecat to also store the name of the department, just like the "Project" field does, save the name when a department is asscoiated with a publication (this is how projects work)
* Add an extra /department/:department_id route to the REST API and side-load the information when a publication is requested.

**Comment by nicolasfranck** [comment]
If you request a publication, the other attributes of a department should be added? That would be ok, only and only if no updates on those attributes are sent back (except for the id). Same is true for other associations like `user_id`, `authors` ..

**Comment by nicolasfranck** [comment]
https://github.com/ugent-library/biblio-backend/pull/69

**Comment by nics** [comment]
resolved by #69
